crates/nu-protocol/src/engine/engine_state.rs

```diff
@@ -713,6 +717,21 @@ impl EngineState {
 
         self.num_files() - 1
     }
+
+    #[cfg(not(windows))]
+    pub fn get_sig_quit(&self) -> &Option<Arc<AtomicBool>> {
+        &self.sig_quit
+    }
+
+    #[cfg(windows)]
+    pub fn get_sig_quit(&self) -> &Option<Arc<AtomicBool>> {
+        None
```

Comment:
```suggestion
        &None
```
50% sure this will fix it

Reply:
It's fixed :-)